src/common/controllers/controller-provider.ts

```diff
@@ -0,0 +1,36 @@
+import { PluginManager } from '../plugins';
```

Comment:
file marked as "created" instead of "renamed", will git save history in such case?